homeassistant/components/image_processing/facebox_face_detect.py

```diff
@@ -0,0 +1,105 @@
+"""
+Component that will perform facial detection via a local facebox instance.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/image_processing.facebox_face_detect
+"""
+import base64
+import requests
+import logging
+import time
+import voluptuous as vol
+
+from homeassistant.core import split_entity_id
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.image_processing import (
+    PLATFORM_SCHEMA, CONF_SOURCE, CONF_ENTITY_ID,
+    CONF_NAME)
+from homeassistant.const import (CONF_IP_ADDRESS, CONF_PORT)
+from homeassistant.components.image_processing.microsoft_face_identify import (
```

Comment:
Why is the base face entity class located in the microsoft platform? This is weird. @pvizeli do you know? Normally we should not rely on another platform in a platform. The base entity class should probably move to `homeassistant.components.image_processing`.

Reply:
@MartinHjelmare I agree and propose a refactor

Reply:
@MartinHjelmare that is correct. On my inital PR it was inside component but I need remove this. In a later discusion about this designe, paulus agree that I can move back the code. But I found never time for this.

Reply:
#robmarkcole if you have time, a refactor in a separate PR would be great.

Reply:
Sure